css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,44 @@
+h1 {
+  text-align: center;
+  font-family: "DM sans", sans-serif;
+}
+
+p {
+  font-family: "DM Sans", sans-serif;
+}
+
+.container {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  padding-left: 80px;
+  padding-right: 80px;
```

Comment:
This should be on a separate class, the `row` class should only have `display: flex` applied to it.